suzieq/config/schema/address.avsc

```diff
@@ -65,7 +65,7 @@
         {
             "name": "vrf",
             "type": "string",
-	    "depends": "master",
+            "depends": "master",
```

Comment:
I don't understand why we have the commits on the schema file in this patch. All these seem like whitespace fixes and they're just different from what was there. We should remove these from this patchset. This obviously excludes the deviceSession field addition

Reply:
In some places in the schema there were tabs instead of the 4 spaces. The editor changed them once I saved. I kept spaces on purpose and I recommend to keep them because every time someone changes the schema that tabs will get replaced.